src/main.cpp

```diff
@@ -110,6 +124,8 @@ int main(int argc, char *argv[]) {
       fprintf(stderr, "\t-c A|B|C|D: Select ftdi channel\n");
       fprintf(stderr, "\t-V VID: Specify usb vid\n");
       fprintf(stderr, "\t-p PID: Specify usb pid\n");
+      fprintf(stderr, "\t-B BUS: Specify usb bus addr\n");
+      fprintf(stderr, "\t-D DEV: Specify usb device addr\n");
```

Comment:
Thanks, we will need some documentation for this in README.

Reply:
Added a line in the documentation.